sagemaker/launch/launch_hyperparameter_tuning.py

```diff
@@ -247,6 +324,19 @@ def get_hpo_parser():
         choices=["Bayesian", "Random", "Hyperband", "Grid"],
         help="Optimization strategy. Default: 'Bayesian'.",
     )
+    # Add new Hyperband-specific arguments
+    hpo_group.add_argument(
+        "--hb-min-epochs",
+        type=int,
+        default=1,
+        help="Minimum number of epochs for Hyperband strategy. Default: 1",
```

Comment:
Shall we point users to https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_HyperbandStrategyConfig.html?

Reply:
Are we only targeting at defining max/min range for epochs? Ideally we should ask users to define the parameters they want to do hyperband themselves.

Reply:
Number of epochs is the most common measure of resource allocation per job, and the one that SM supports out-of-the box. Note that users are still able to optimize any parameter they want as usual, all Hyperband does is stop non-promising jobs early to allocate resources to the more promising ones, and `--hb-max-epochs` only controls the maximum number of epochs Hyperband will allow a job to run for.

See https://docs.aws.amazon.com/sagemaker/latest/dg/automatic-model-tuning-how-it-works.html#automatic-tuning-hyperband

Reply:
@classicsong I'll add a note to the documentation for that.